application/templates/backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: {{ .Values.backup.backupCRName }}
```

Comment:
@ayeshasiddiqa161 can't this be dynamic?

Reply:
Yes working on it.

Reply:
Resolved